dev-tools/ecs-migration.yml

```diff
@@ -6,44 +6,46 @@
 #
 # - from: source-field-in-6.x
 #   to: target-filed-in-ECS
-#   # Alias field is useful for fields where there is a 1-1 mapping from old to new
-#   alias: true-if-alias-is-required-in-6x
+#   # Alias field is useful for fields where many-1 mapping from new to old are needed
+#   alias: true-if-alias-is-required-in-7
+#   # Alias6 field is useful for fields where there is a 1-1 mapping from old to new. If absent, defaults to false.
+#   alias6: true-if-alias-is-required-in-6
 #   # Copy to is useful for fields where multiple fields map to the same ECS field
 #   copy_to: true-if-field-should-be-copied-to-target-in-6x
 
 - from: offset
   to: log.offset
+  alias6: true
   alias: true
-  copy_to: false
 
 - from: fileset.name
   to: event.dataset
+  alias6: true
   alias: true
-  copy_to: false
 
 - from: fileset.module
   to: event.module
+  alias6: true
   alias: true
 
 - from: source
   to: ["log.file.path", "log.source.ip"]
   alias: false
-  copy_to: false
 
 - from: beat.name
   to: agent.type
+  alias6: true
   alias: true
-  copy_to: false
 
 - from: beat.hostname
   to: agent.hostname
```

Comment:
@MikePaquette is pushing back on `agent.hostname`, btw.

Agent should always be running on the host itself. If the monitoring agent is outside of the host, it should be represented in `observer.hostname` (used to be called `device.hostname`, but device is being renamed to observer).

Reply:
Should be addressed in another PR, though.